type/global.d.ts

```
@@ -0,0 +1,220 @@
+        url: string;
+        datetime: number;
+        category: string;
+        related: string;
+        image?: string;
+    };
+
+    type WatchlistNewsProps = {
+        news?: MarketNewsArticle[];
+    };
+
+    type SearchCommandProps = {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'SearchCommandProps'. Consider to delete it or rename it.

'SearchCommandProps' is defined here:


(lint/suspicious/noRedeclare)